assets/js/push.js

```diff
@@ -581,6 +581,11 @@ jQuery( window ).on( 'load', () => {
 
 				delete selectedConnections[ type + id ];
 
+				document.querySelector(
+					'.selected-connections-count'
+				).textContent =
+					'( ' + Object.keys( selectedConnections ).length + ' )';
```

Comment:
```suggestion
					'(' + Object.keys( selectedConnections ).length + ')';
```

Consistent presentation